.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
- [ ] [OPTIONAL] Did you mean to track this file in git? Because it has been ignored in your `.gitignore` file but it is still tracked.

Reply:
Thank you. Fixed

Reply:
Well resolved this!